engine/src/main/java/com/core/rpgmaker/RpgMakerActivity.kt

```
@@ -0,0 +1,954 @@
+    /** 影响引擎行为的 Intent extras 摘要，用于判断单游戏重启是否需要重建。 */
+    private fun behaviorSignature(intent: Intent): String = listOf(
+        resolveGameDir(intent),
+        intent.getBooleanExtra(EXTRA_SCOPED_SAVE_DIR, false).toString(),
+        intent.getStringExtra(EXTRA_SCOPED_SAVE_ROOT),
+        intent.getBooleanExtra(EXTRA_RPG_MAKER_MOD_ENABLED, true).toString(),
+        intent.getStringExtra(EXTRA_RPG_MAKER_MOD_GAME_ID),
+        intent.getStringExtra(EXTRA_RPG_MAKER_VERSION),
+        intent.getBooleanExtra(EXTRA_RPG_LEGACY_RENDERER, false).toString(),
+    ).joinToString("\u0000")
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

detekt (1.23.8)

[warning] The function onCreate appears to be too complex based on Cyclomatic Complexity (complexity: 45). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)